draft-ietf-acme-acme.md

```diff
@@ -863,6 +872,32 @@ label) MUST NOT be included in authorization requests.
 }
 ~~~~~~~~~~
 
+## Getting a Nonce
+
+Before sending a POST request to the server, an ACME client needs to have a
+fresh anti-replay nonce to put in the "nonce" header of the JWS.  In most cases,
+the client will have gotten a nonce from a previous request.  However, the
+client might sometimes need to get a new nonce, e.g., on its first request to
+the server or if an existing nonce is no longer valid.
+
+To get a fresh nonce, the client sends a HEAD request to the new-nonce resource
+on the server.  The server server's resposne MUST include a Replay-Nonce header
+field containing a fresh nonce, and SHOULD have status code 200 (OK).  The
+server SHOULD also respond to GET requests for this resource, returning an empty
+body.
+
+~~~~~~~~~~
+HEAD /acme/new-nonce HTTP/1.1
+Host: example.com
+
+HTTP/1.1 200 OK
+Replay-Nonce: oFvnlFP1wIhRlYS2jTaXbA
+Cache-Control: no-store
+~~~~~~~~~~
+
+The new-nonce resource MUST NOT be cached by the client or intermediate HTTP
+caches.  The server SHOULD include a Cache-Control header field with the
```

Comment:
Caching can basically induce a DOS (inability to get a fresh nonce), but I see your point w.r.t. the client/cache side requirement.  Deleting that and upgrading the server requirement to MUST.